### ISSUE TITLE
Allow other names for `plantuml.jar`

### ISSUE THREAD

**Issue by xbreak** [doxygen/doxygen#8965]
**Describe the bug**

Currently Doxygen only supports PlantUML if the .jar is named ["plantuml.jar"](https://github.com/doxygen/doxygen/blob/f62ecdf7adcfdafd1b52320fb867495fada63e79/src/plantuml.cpp#L174). This cause unnecessary problems with distributions that do not name the jar exactly like that. Anaconda distribution for example use the full version in the name which results in the path `/opt/anaconda3/lib/plantuml.1.2021.1.jar`. 

Example configuration:

```
...
PLANTUML_JAR_PATH      = /opt/anaconda3/lib/plantuml.1.2021.1.jar
```
yields error:
```
...
Running plantuml with JAVA...
Generating PlantUML png Files in html
Error: Unable to access jarfile /opt/anaconda3/lib/plantuml.jar
error: Problems running PlantUML. Verify that the command 'java -jar "/opt/anaconda3/lib/plantuml.jar" -h' works from the command line. Exit code: 1
```

**Expected behavior**

If the path to the plantuml jar-file is specified I'd expect Doxygen to use that verbatim.

**To Reproduce**
Any example using plantuml with that is not named "plantuml.jar" should work. Attached is a minimal example that use anaconda JAR:

[example.tar.gz](https://github.com/doxygen/doxygen/files/7704800/example.tar.gz)

**Version**
v1.9.1 (nixpkgs)
v1.8.20 (CentOS 8)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8967

**Comment by xbreak** [comment]
Thanks for the quick response! Proposed fix looks good to me.

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.3.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
